public-ui/src/app/data-browser/views/survey-chart/survey-chart.component.html

```diff
@@ -17,6 +17,7 @@
   </div>
 </div>
 <div class="chart" *ngIf="!displayGraphErrorMessage">
+  <app-biosex-chart-react *ngIf="isBioSexChart()" (click)="$event.stopPropagation()" [domain]="'survey'" [genderAnalysis]="selectedChartAnalysis" [genderCountAnalysis]="surveyCountAnalysis.genderCountAnalysis" [selectedResult]="selectedResult"></app-biosex-chart-react>
   <app-chart (click)="$event.stopPropagation()" [surveyAnalysis]="selectedChartAnalysis" [selectedResult]="selectedResult" [surveyCountAnalysis]="surveyCountAnalysis"
```

Comment:
Should there be a condition on the Angular version? Currently both charts are getting displayed:
![localhost_4201_survey_the-basics](https://user-images.githubusercontent.com/40036095/111727195-693bb980-8838-11eb-9ac0-75c3425941ce.png)

Reply:
Added a condition in here. I just left it out to make it easier for comparison.

Reply:
![image](https://user-images.githubusercontent.com/21062724/111814081-0a158d80-88a8-11eb-9a66-12aee66490bc.png)